### PR TITLE
Round calculator estimates to the nearest whole number

### LINKED ISSUE #458: Always show whole numbers on calculators
Requested by UX, seems to simplify the view.

### PR DESCRIPTION
**Pull request by benbarnett**
Closes #458.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module ApplicationHelper
   include NavigationHelper
 
   def format_currency(amount, **options)
-    number_to_currency(amount, options)&.sub(/\.00$/, '')
+    number_to_currency(amount, options.reverse_update(precision: 0))&.sub(/\.00$/, '')
```
Comment (andrewgarner) [inline app/helpers/application_helper.rb]: I think you can set precision in the `en.yml` file.

`&.sub(/\.00$/, '')` is also now redundant.

Comment (benbarnett) [inline app/helpers/application_helper.rb]: Agree. Might we want this behaviour at some point though?

Comment (andrewgarner) [inline app/helpers/application_helper.rb]: We can re-introduce it if we need it.

Comment (benbarnett) [inline app/helpers/application_helper.rb]: Cool. Have cleaned it up now.

**Comment by andrewgarner** [comment]
:+1:

**Comment by andrewgarner** [comment]
Actually ignore my thumb - one more comment

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module ApplicationHelper
   include NavigationHelper
 
   def format_currency(amount, **options)
-    number_to_currency(amount, options)&.sub(/\.00$/, '')
+    number_to_currency(amount, options)
```
Comment (andrewgarner) [inline app/helpers/application_helper.rb]: We might as well remove `format_currency` completely as all it now does is proxy to `number_to_currency`. We could just use `number_to_currency` in our views and keep things closer to the Rails norm.

**Comment by andrewgarner** [comment]
:+1:

**Comment by benbarnett** [comment]
Ta. Much better.

**Pull request merged**